resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -75,6 +75,12 @@ private[spark] object Config extends Logging {
       .toSequence
       .createWithDefault(Nil)
 
+  val SUBMISSION_CLIENT_PREFIX =
+      "spark.kubernetes.kubernetesClient.submission"
+  val DRIVER_CLIENT_PREFIX =
+      "spark.kubernetes.kubernetesClient.driver"
+  val CLIENT_MODE_CLIENT_PREFIX =
+      "spark.kubernetes.kubernetesClient"
```

Comment:
Hi, @onursatici . Since these configurations have the default values, please use `ConfigBuilder` like line 70 ~ 76.

Reply:
ok will do

Reply:
Ur, please remove these lines and put the string to `ConfigBuilder`s. After that, we use `SUBMISSION_CLIENT_REQUEST_TIMEOUT.key` .